src/main/drivers/vtx_smartaudio.c

```diff
@@ -0,0 +1,706 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <ctype.h>
+
+#include "platform.h"
+
+#ifdef VTX_SMARTAUDIO
+
+#include "drivers/system.h"
+#include "drivers/serial.h"
+#include "drivers/vtx_smartaudio.h"
+#include "io/serial.h"
+#include "fc/runtime_config.h"
+#include "config/config_master.h"
```

Comment:
Drivers should not call upwards, so should not include any of:
`io/serial.h`
`fc/runtime_config.h`
`config/config_master.h`